### ISSUE TITLE
configure: error: Required cups-config is missing

### ISSUE THREAD

**Issue by andyrtr** [OpenPrinting/libppd#10]
libppd 2.0b3 is checking for cups-config that is not (yet?) part of new libcups 3.0b1 so the build fails here for Arch Linux:
```
checking for cups-config... no
configure: error: Required cups-config is missing. Please install CUPS developer packages.
```

What's the recommended package versions of libcups / cups-server here now?

**Comment by zdohnal** [comment]
cups-config is deprecated in CUPS 2.x and removed in CUPS 3.x - configure.ac-s of OpenPrinting projects will have to be adjusted to work with pkgconfig, which support was added to CUPS in 2.4.x series.

Otherwise required CUPS version is 2.2.2 as it is mentioned in INSTALL.

**Comment by andyrtr** [comment]
Thanks, leaving out libcups 3.0b1 here and using former libcups 2.4.2 made the build finish.

**Comment by tillkamppeter** [comment]
libcups 3.x support is in the works all components of cups-filters 2.x. The support for libcups 3.x is nearly completed in libcupsfilters, see https://github.com/OpenPrinting/libcupsfilters/pull/24

The libcups 3.x support will be part of the 2.1.0 releases. We do not yet add it into 2.0.x as we are shortly before the Ubuntu 23.04 release where we want to include the most recent fixes as version 2.0.0 and not risk any new bugs by the higher-impact change of adding libcups 3.x support.

**Comment by tillkamppeter** [comment]
libcups3 support is in place now, at least in the GIT master branches, of libcupsfilters 2.x, libppd 2.x, pappl 1.x, pappl-retrofit. In pappl 2.x libcups3 is required. So taking the GIT snapshots the libraries should work with libcups3 now.

So I am closing this bug report.
